packages/react-devtools-core/src/standalone.js

```diff
@@ -43,6 +43,11 @@ let nodeWaitingToConnectHTML: string = '';
 let projectRoots: Array<string> = [];
 let statusListener: StatusListener = (message: string) => {};
 
+// TODO (Webpack 5) Hopefully we can remove this prop after the Webpack 5 migration.
+function hookNamesModuleLoaderFunction() {
+  return import('./hookNames');
```

Comment:
Do we need a comment like the one on #22322 to avoid a chunk name like 0.x? Or is the `hookNames` entrypoint enough to avoid this?

Reply:
oh I wasn't aware of that actually, let me try. in #22322, why did we not also have to update the webpack config / loader function for the inline package?

Reply:
so without this comment, i do see a `hookNames.js` bundle in `dist/`, but i also see a `2.chunk.js` file which also seems to contain parseHooks code:

![image](https://user-images.githubusercontent.com/1271509/133489424-b623fba3-8aec-46fa-9b9d-cbbf94d3af8c.png)

not sure if this is what we want, so I think i'll try to do the same thing extensions is doing

Reply:
alright, I changed it to do the same setup we do with extensions and seems to be working better now:

![image](https://user-images.githubusercontent.com/1271509/133490752-dc60431f-02d2-4a0e-bd08-9126d1733d2c.png)